### ISSUE TITLE
Issue with shallow rendering in React 16/Enzyme 3.2

### ISSUE THREAD

**Issue by af** [glortho/react-keydown#76]
While migrating to React 16, we noticed that some of our Enzyme shallow rendering tests are no longer working with `react-keydown`. We get errors from various components that all follow the following pattern:

```
    Invariant Violation: Unable to find node on an unmounted component.

      at invariant (node_modules/fbjs/lib/invariant.js:42:15)
      at Object.findDOMNode (node_modules/react-dom/cjs/react-dom.development.js:15286:7)
      at Object.bindFocusables (node_modules/react-keydown/dist/lib/dom_helpers.js:34:35)
      at onMount (node_modules/react-keydown/dist/event_handlers.js:104:25)
      at DatePicker.target.componentDidMount (node_modules/react-keydown/dist/decorators/method_decorator.js:59:35)
      at node_modules/enzyme/build/ShallowWrapper.js:126:20
      at Object.batchedUpdates (node_modules/enzyme-adapter-react-16/build/ReactSixteenAdapter.js:344:22)
      at new ShallowWrapper (node_modules/enzyme/build/ShallowWrapper.js:125:24)
      at shallow (node_modules/enzyme/build/shallow.js:19:10)
      ...
```

Our usage of `react-keydown` is as follows (decorators on methods of class components):

```js
	@keydown('esc')
	_handleClose () {
		this.props.onClose();
	}
```

Converting these tests to use Enzyme's `mount()` resolves the errors, but the shallow rendering tests were working on React 16 and it'd be nice to keep them that way if possible. The traceback leads to `react-keydown`'s use of `findDOMNode()` in `bindFocusables`, if that helps.

versions:
React: 16.1.1
enzyme: 3.2.0
enzyme-adapter-react-16: 1.1.0
react-keydown: 	1.9.5

**Comment by glortho** [comment]
Okay it should be easy enough to suppress that during shallow rendering. We're on tight deadline at the moment, but I'll see if I push up a fix in the next couple of days.

**Comment by af** [comment]
Great, thanks!

**Comment by glortho** [comment]
I've published 1.9.6 with error catching to prevent that from throwing but there is obviously a deeper issue to address here. If the catching doesn't resolve it for you @af I can investigate more in a couple weeks. Let me know!

**Comment by af** [comment]
This fixes it, thank you!
